spec/cassettes/Identity/when_sending_the_trn_to_the_Identity_API/redirects_back_to_Get_An_Identity.yml

```diff
@@ -37,31 +37,31 @@ http_interactions:
         X-Rate-Limit-Limit:
           - 60s
         X-Rate-Limit-Remaining:
-          - "298"
+          - "299"
         X-Rate-Limit-Reset:
-          - "2022-08-19T16:07:00.0000000Z"
+          - "2022-08-24T10:38:00.0000000Z"
         X-Vcap-Request-Id:
-          - 84caf59e-b10f-49e3-6d1d-eda5c5b58583
+          - 2dc630de-12c4-4c0d-7309-e4790413b478
         X-Xss-Protection:
           - "0"
         X-Cache:
           - Miss from cloudfront
         Via:
-          - 1.1 08dc6f02f30e8ad9291872e7e3d5b658.cloudfront.net (CloudFront)
+          - 1.1 22e421a47e59010b5e8eb6ae4d4bd7e4.cloudfront.net (CloudFront)
         X-Amz-Cf-Pop:
           - LHR61-P2
         X-Amz-Cf-Id:
-          - 0IeN4mTQHrR7VnVwGrUgCH6TiQ0heWSbREGMMfNAybKqczCZNyyEig==
+          - Wiwy47h4WPdk7jToS7sBfkdOfFFqKv6P_ejsEHC1XFIv_mgum8coHA==
       body:
         encoding: UTF-8
         string: '{"results":[{"trn":"2921020","emailAddresses":["kevin.e@example.com"],"firstName":"Kevin","lastName":"E","dateOfBirth":"1990-01-01","nationalInsuranceNumber":"AA123456A","uid":"1bf8803f-3924-417c-9603-1ed6489dae0d","hasActiveSanctions":false}]}'
-    recorded_at: Fri, 19 Aug 2022 16:06:59 GMT
+    recorded_at: Wed, 24 Aug 2022 10:37:55 GMT
   - request:
       method: put
       uri: https://s165d01-getanid-dev-auth-server-app.azurewebsites.net/api/find-trn/user/9ddccb62-ec13-4ea7-a163-c058a19b8222
       body:
         encoding: UTF-8
-        string: '{"firstName":"Kevin","lastName":"E","trn":"2921020","dateOfBirth":"1990-01-01"}'
+        string: '{"firstName":"Kevin","lastName":"E","dateOfBirth":"1990-01-01"}'
```

Comment:
Hmm, why was the TRN being included in the request body previously?

Reply:
The request is one time only, it will error if the same request is sent multiple times for the same Identity journey id. To get around this I added the request into necessary cassettes and must have copied the no trn request. I have now added it back in